housing_django/static/js/applicantcreationform.js

```diff
@@ -0,0 +1,40 @@
+var ApplicantCreationForm = Backbone.View.extend({ 
+  initialize: function() {
+    this.template = Handlebars.compile($("#applicant-form-tmp").html());
+  },
+  events: {
+    'click .btn-primary': 'submitForm'
```

Comment:
Can you add a comment here saying that you should use submit, but are using click because submit is not working? This way someone who might have a better handle on Backbone can help you fix in the future.